frontend/src/app/patients/PatientForm.tsx

```diff
@@ -287,7 +303,7 @@ const PatientForm = (props: Props) => {
             required
           />
           <TextInput
-            label="Middle Name"
+            label="Middle Name (optional)"
```

Comment:
I'd inadvertently removed this earlier, it's still in the engineering prototype.